modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -39,6 +39,107 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-176607. Verify that is not possible to edit an organization information without EVP Manager role"
+	@priority = 5
+	test CanNotEditWithoutManagerRole {
+		task ("Given I have created a request for a new Organization") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "employee@liferay.com",
+				userLoginFullName = "Employee User");
+
+			EVPUtils.gotoEVP();
+
+			EVPNavigator.goToNavItem(navItem = "Organization");
+
+			Button.click(button = "Create an Organization");
+
+			EVPOrganization.fillOrganizationForm(
+				bankingInfo = "12345-6",
+				city = "Organization City",
+				contactEmail = "contact@organization.com",
+				contactName = "Contact Name",
+				contactPhone = "12 345678901",
+				country = "Country Site",
+				organizationName = "Organization Test",
+				organizationSite = "organization.com",
+				smallDescription = "Small Description",
+				state = "Organization State",
+				street = "Organization Street",
+				taxId = 12345,
+				zipCode = "12345-678");
+
+			EVPNavigator.goToNavItem(navItem = "Organization");
+		}
+
+		task ("And the Organization is aprooved by Manager and Finance Users") {
```

Comment:
Oh i got it. Will look better if you should split in two test. one for user role and other one to finance role.

Reply:
The Organization has to pass both EVP Manager User and Finance user approvals. 
Not sure why it would be better to split the test in two. Can you elaborate?